src/main/scala/AI.scala

```diff
@@ -0,0 +1,23 @@
+
+
+object AI {
+
+  def moveAI( Str: String) : String = {
+    val randomAI = new scala.util.Random
+    val randomNum = randomAI.nextInt(2)
+    var choiceAi  = " "
+    if (randomNum == 0 )(choiceAi = "rock")
+    if (randomNum == 1 ) (choiceAi = "scissors")
+    if (randomNum == 2 ) (choiceAi = "paper")
+
+    return choiceAi
```

Comment:
you don't need the keyword return here

Reply:
you take in an argument but don't use it, why is it there? the whole way of implementation could be done like this in a more efficient way:

def moveAI = {
  val randomAI = new scala.util.Random
  randomAI.nextInt(2) match {
    case 0 => "rock"
    case 1 => "scissors"
    case 2 => "paper"
  }
}